Sources/Testing/StressTestCase.swift

```diff
@@ -6,24 +6,7 @@
 
 import Foundation
 import XCTest
-
-public class Counter {
-    private(set) var _count: Int32 = 0 // swiftlint:disable:this variable_name
-
-    public var count: Int {
-        return Int(_count)
-    }
-
-    @discardableResult public func increment() -> Int32 {
-        return OSAtomicIncrement32(&_count)
-    }
-
-    @discardableResult public func barrierIncrement() -> Int32 {
-        return OSAtomicIncrement32Barrier(&_count)
-    }
-
-    public init() { }
-}
+@testable import ProcedureKit
```

Comment:
This is a bit of a problem - it means that we can't create Release configuration builds of `TestingProcedureKit`, so, for example integration via Carthage will fail.

I'm guessing it's the `.withCriticalScope { }` API that you need? I have no issue making that public.